### PR TITLE
Add market ID to market update proposal

### LINKED ISSUE #297: Add UpdateMarket proto to ProposalSubmission
Linked to:
- https://github.com/vegaprotocol/vega/issues/4704

### PR DESCRIPTION
**Pull request by ValentinTrinque**
Close #297

### CODE REVIEW

**Comment by jeremyletang** [comment]
There must be an issue here. How come this updated so much files?

File: sources/vega/governance.proto [hunk 1]
```diff
@@ -86,8 +86,10 @@ message NewMarket {
 
 // Update an existing market on Vega
 message UpdateMarket {
+  // The market to update
+  string market_id = 1;
   // The updated configuration of the market
-  UpdateMarketConfiguration changes = 1;
+  UpdateMarketConfiguration changes = 2;
 }
```
Comment (jeremyletang) [inline sources/vega/governance.proto:93]: Why not add it inside the changes directly?

**Comment by wwestgarth** [comment]
> There must be an issue here. How come this updated so much files?

I spotted that too. Looking in the history it seems some recent data-node changes were merged it only having generated the grpc bits, and not the swagger code.

File: sources/vega/governance.proto (hunk 1 above)
Comment (ValentinTrinque) [reply sources/vega/governance.proto:93]: I wanted to keep the market configuration focus on the actual market change, and not the additional data that help to retrieve the market

**Comment by ValentinTrinque** [comment]
I moved the additional JSON file to another PR.

**Review by peterbarrow — APPROVED** [review]
The changes to the jenkins jobs look great, thanks Valentin for sorting.

**Pull request merged**